live-ingester/src/main/resources/application.yml

```diff
@@ -14,28 +14,26 @@ hyland:
 
 alfresco:
   integration:
+    repository:
+      endpoint: activemq:topic:alfresco.repo.event2
```

Comment:
This change makes it clearer what the property is for :+1:

Reply:
Definitely I agree